### DISCUSSION TITLE
Code indentation in org files

### DISCUSSION THREAD

**Discussion by andreadev-it** [nvim-orgmode/orgmode#440] [General]
Hi everyone, I've just started a discussion to understand what options do I have for my current situation, and thought it might be helpful for others too.

I've moved all my nvim config to an org config file (multiple files, for now) but I'm having some trouble when I need to write or edit lua code inside the code blocks. The problem is that my indentation settings do not take into account org's indentation, so every time I write something the number of spaces always gets messed up.

I was wondering how this was handled in Emacs, and I've seen that no lines are indented at all, it's emacs that indents them visually. This is helpful because it doesn't create problem with the actual number of whitespaces in the code, but I don't think it is possible in neovim. 
I'm thinking, if it's possible, to disable org indentation in my plugin settings and see how it feels. How are you handling this (if you had a similar issue)?

**Comment by jgollenz** [comment]
Sorry for the late response. Could you perhaps add a short example? I'm not sure if I completely understand. It sounds like your issue is related to #422

**Reply by andreadev-it** [reply]
Hi @jgollenz, yes, that is indeed the solution I was looking for. Since that functionality is not yet available, I was wondering how others are doing code indentation in orgmode

**Reply by jgollenz** [reply]
I'm actually (since I didn't even know about `org-indent-mode` when I started with nvim-orgmode) just manually indenting it with whitespaces :sweat_smile:

**Reply by cdbzb** [reply]
writing to subscribe. this is most annoying! hoping to use orgmode with supercollider code blocks -- syntax highlighting is great but very hard to use without indentation inside of code blocks!
